x-pack/plugins/security_solution/cypress/integration/timeline_template_creation.spec.ts

```diff
@@ -75,9 +71,6 @@ describe.skip('Timeline Templates', () => {
     closeNotes();
     markAsFavorite();
     waitForTimelineChanges();
-    createNewTimelineTemplate();
```

Comment:
Why are we deleting these lines? The idea of this flow, is to check that once the timeline is created you can open it again and see that the content is the proper one. So with the `createNewTimelineTemplate();` method we are wiping out the content of the flyout, and with `closeTimeline();`	`openTimelineFromSettings();` we are opening the created timeline again. If we remove these lines we are not completely checking that after the creation of the timeline, you can properly retrieve it,